src/main/docker/Dockerfile.jdk

```diff
@@ -24,12 +24,18 @@ ADD https://raw.githubusercontent.com/CAFapi/caf-common/v1.14.0/container-cert-s
     /startup/startup.d/
 RUN chmod +x /startup/startup.d/install-ca-cert-java.sh
 
-# Install OpenJDK 8 Development Kit
-RUN zypper -n install java-1_8_0-openjdk-devel
+# Install OpenJDK 8 Development Kit and Patch
+RUN zypper -n install java-1_8_0-openjdk-devel patch
 
 # Set Java Home
 ENV JAVA_HOME=/usr/lib64/jvm/java-1.8.0-openjdk-1.8.0
 
+# Disable weaker security algorythms
+COPY 0001-prevent-Logjam-disableAlgorythms.patch $JAVA_HOME/jre/lib/security/
+RUN cd $JAVA_HOME/jre/lib/security && \
+    patch -i 0001-prevent-Logjam-disableAlgorythms.patch java.security && \
+    rm 0001-prevent-Logjam-disableAlgorythms.patch
```

Comment:
I notice this is leaving behind a `java.security.orig` file.  I know you could just delete it but let's do this in a builder image instead.  It will save doing the cleanup but more importantly it will set us up better for future configuration tinkering.

Reply:
On the other side, keeping of original `java.security` would allow reverting to default in derived images, if required.

Reply:
OK fair point.  Leave the `java.security.orig` file then.  Although still do the patching in a builder image for other reasons like keeping the number of layers down and not installing the `patch` utility (which is unlikely to be required by a running application).

Reply:
Done